api/python/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+[package]
+name = "pyslint"
```

Comment:
If that was just me, i'd name this "slint-python",  but that's detail

Reply:
Yeah, we can (and should) refine that name before publishing.

Reply:
Renamed now to "slint-python" :) (well, in next push)